Sources/Plasma/PubUtilLib/plFile/CMakeLists.txt

```diff
@@ -23,6 +23,7 @@ target_link_libraries(
     PUBLIC
         CoreLib
     PRIVATE
+        plNetClient
```

Comment:
`plNetClient` isn't being used anywhere.

Reply:
Yeah. Something is weird here - plNetClient worked as the dependency locally but it's not on Actions. But it's not the right dependency - and if we removing the logging entirely this won't be necessary.

Reply:
I think the linker errors are because the use of `plNetApp` is indirectly pulling in all the infrastructure around `plCreatable`, and some executables (`plFileEncrypt` and `plUruLauncher`) that use `plFile` don't have the magic "AllCreatables" includes that are needed to make creatables work.

(I'm a little surprised though. Did the launcher really not use *any* creatables, keyed objects, or messages before?)